curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+Which CSS property controls the spacing between lines of text?
 
 #### --distractors--
 
-Placeholder distractor 1
+`letter-spacing`
 
 ---
 
-Placeholder distractor 2
+`line-spacing`
 
 ---
 
-Placeholder distractor 3
+`word-spacing`
 
 #### --answer--
 
-Placeholder answer
+`line-height`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common use for `aria-hidden="true"`?
 
 #### --distractors--
 
-Placeholder distractor 1
+To show content
 
 ---
 
-Placeholder distractor 2
+To display on mobile
 
 ---
 
-Placeholder distractor 3
+To increase visibility
 
 #### --answer--
 
-Placeholder answer
+To hide content from screen readers
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the `flex` property used for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changing text color
 
 ---
 
-Placeholder distractor 2
+Adjusting font size
 
 ---
 
-Placeholder distractor 3
+Adding animations
 
 #### --answer--
 
-Placeholder answer
+Creating responsive layouts
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a CSS Grid property?
 
 #### --distractors--
 
-Placeholder distractor 1
+`flex-direction`
 
 ---
 
-Placeholder distractor 2
+`float`
 
 ---
 
-Placeholder distractor 3
+`position`
 
 #### --answer--
 
-Placeholder answer
+`grid-area`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does CSS stand for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Creative Style Sheets
 
 ---
 
-Placeholder distractor 2
+Colorful Style Sheets
 
 ---
 
-Placeholder distractor 3
+Computer Style Sheets
 
 #### --answer--
 
-Placeholder answer
+Cascading Style Sheets
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the minimum contrast ratio for graphics?
 
 #### --distractors--
 
-Placeholder distractor 1
+1.5:1
 
 ---
 
-Placeholder distractor 2
+4.5:1
 
 ---
 
-Placeholder distractor 3
+3:1
 
 #### --answer--
 
-Placeholder answer
+3:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property helps with responsive design?
 
 #### --distractors--
 
-Placeholder distractor 1
+`position`
 
 ---
 
-Placeholder distractor 2
+`width`
 
 ---
 
-Placeholder distractor 3
+`overflow`
 
 #### --answer--
 
-Placeholder answer
+`max-width`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property aligns text in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+`align-text`
 
 ---
 
-Placeholder distractor 2
+`justify-text`
 
 ---
 
-Placeholder distractor 3
+`text-position`
 
 #### --answer--
 
-Placeholder answer
+`text-align`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `outline` property affect?
 
 #### --distractors--
 
-Placeholder distractor 1
+Border of an element
 
 ---
 
-Placeholder distractor 2
+Margin of an element
 
 ---
 
-Placeholder distractor 3
+Padding of an element
 
 #### --answer--
 
-Placeholder answer
+Focus indicator around an element
```

Comment:
This question is wrong and I swear I mentioned this at some point. But the `outline` property only controls the border around an element. Not a focus indicator. 

https://developer.mozilla.org/en-US/docs/Web/CSS/outline

If it is used in conjunction with the `:focus` pseudo-class selector, maybe I can see it. But this is not what this question is about.

Reply:
> Assigning outline a value of 0 or none will remove the browser's default focus style.

I do get where they are coming from, tbh.

Reply:
But at the same time, having `border of an element` listed as an incorrect answer when `outline` can be used to control an element's border makes the question invalid.

Reply:
I have no problem with how this question and answers are worded. The outline is not the same as the border. Outlines are almost exclusively used for focus indicators (every browser uses the outline as the focus indicator by default). And the vast majority of accessibility professionals consider the outline to be the focus indicator and will warn you to never set `outline: none`.

But if there are worries about the way this question and answers are worded, then one suggestion is to rephrase the question to something like:

"What is the `outline` property most commonly used to configure?"